### ISSUE TITLE
Adding devices to Insteon 

### ISSUE THREAD

**Issue by jrwitte** [home-assistant/home-assistant.github.io#22498]
### Feedback

Reads like devices will be automatically discovered but I'm only able to get the hub added.  Couldn't find any obvious direction on how to add a device.

### URL

https://www.home-assistant.io/integrations/insteon/

### Version

2022.4.6

**Comment by klaasnicolaas** [comment]
https://www.home-assistant.io/integrations/insteon/#adding-devices-to-the-insteon-integration

**Comment by jrwitte** [comment]
Been there. Don’t know what “All-Link group number” is.  If I just pick arbitrary “1” Call Service turns green then back to blue and nothing happens even after I pressed the set button on the hub.

**Comment by jrwitte** [comment]
More background:  hub and all devices have been factory reset. Insteon add-on for HA finds only hub. Trying to understand how to find other devices. Tried to print_all but it says no entities to print.

**Comment by redand65** [comment]
i am in the same boat. i tried add all link,  not knowing what group number to use i tried 255 thinking it might be some sort of broadcast. 2 devices went from red to green when i reset them but no new devices show up in the gui.

**Comment by jrwitte** [comment]
Fixed the problem in two instances.  First time was finding a posting that I cannot recall now.  When you run the service to add devices it blinks green then back to blue.  At that point you need to go to the device and hold the set button until it beeps.  The device then shows up.  Had to do this for each device.  This was when I had installed Home Assistant on my Ubuntu laptop.  The laptop install was a bad idea as the operation was not reliable.  I subsequently moved it to a Raspberry Pi 3 Model B+.  Seems to work a lot better there and managed to find most devices on its own.  It has been completely reliable since.

**Comment by jrwitte** [comment]
One more note - found the link.   https://community.home-assistant.io/t/adding-insteon-devices/364188

**Comment by pjkerly** [comment]
New to Home Assistant ... as a result of Insteon going away.  I've spent a few days configuring the system.  I have a mix of Insteon switches collected over the years.  I was able to auto discovery all but one of my switches and all of my leak sensors ... except one SwitchLinc 2476D (very frustrating).  From the insteon devices and all link tables, I could tell that it was communicating with the hub but did not complete the configuration like the others that worked.  I started to remove it from it's current location and install/replace another one that worked flawlessly wondering if there was a signaling issue.  I noticed that the one that would not show up was V5.0 but the ones that worked where all V5.1 or later.  I never paid attention when I installed them originally but had several still sitting around that were versions 5.1.  I replaced it and the unit was auto discovered.  I noticed in Insteon developer documentation that they had made changes to the Insteon protocol from i1, i2, and i2cs (from memory).  I'm wondering if some of the issues folks are having are related to older Insteon versions.  Is there a compiled listed of Insteon devices (and their versions) that are known to work?  If not, maybe it is worth looking into.  Also it would be great if there was a noticed made when a device is detected which is not using the newer protocol?  Not sure if it is worth going back to implement the older protocols?  Just food for thought for anyone running into non-detected.  I have a 2014 Smart Labs Rev Control list of devices [https://usermanual.wiki/m/bd75616a3e3ba3c1d25c554b79bac836f792edfcbdbfe96903a0ca1c3fc0deda.pdf]
